packages/wdio-browserstack-service/src/service.js

```diff
@@ -57,32 +67,51 @@ export default class BrowserstackService {
             `${test.parent} - ${test.title}`
         )
 
-        if (!results.passed) {
-            this.failures++
-            this.failReason = (test.error && test.error.message ? test.error.message : 'Unknown Error')
```

Comment:
@christian-bromann can you confirm this change for `afterTest`?
`test.error` ->`error` from las parameter object

Reply:
yes, this is the v6 way of doing things. 👍